src/js/client/helpers.js

```diff
@@ -0,0 +1,196 @@
+/* global Handlebars, editPrefs */
+"use strict";
+
+/**
+ * i18n Messages
+ *
+ * This is a prototype implemention of our handlebars i18n message handler
+ * to suffice until our work in GPII-2444 lands. https://issues.gpii.net/browse/GPII-2444
+ */
+Handlebars.registerHelper("message", function (messageKey) {
+    var messages = {
```

Comment:
Public data structure please

Reply:
Everything is using the legit gpii-handlebars messageHelper now